lib/evmone/eof.cpp

```diff
@@ -0,0 +1,165 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2020 The evmone Authors.
```

Comment:
Nit: is this 2020? :)

Reply:
It was created in 2021, so can be changed to that :)